docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,700 @@
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+com.uid2.UID2Manager.Companion#init
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+see UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/Info.plist
+```
+
+</TabItem>
+</Tabs>
+
+If necessary, you can also change the default Subscription ID and public key to ones assigned to you, and connect to the UID2 Production environment. For details, see [Optional: Reduce Latency by Setting the API Base URL for the Production Environment](#optional-reduce-latency-by-setting-the-api-base-url-for-the-production-environment).
+
+## Complete the UID2 Account Setup
+
+To set up your account, follow the steps described in [Account Setup](../getting-started/gs-account-setup.md). As part of the account setup process, you'll need to provide a list of <Link href="../ref-info/glossary-uid#gl-app-name">app names</Link> for all the mobile apps that you'll be integrating with the UID2 mobile SDKs, including any of these values that apply:
+
+- Android Application ID
+- iOS App Bundle ID
+- App Store ID
+
+When account setup is complete, you'll receive a [Subscription ID and public key](../getting-started/gs-credentials.md#subscription-id-and-public-key). These values are unique to you, and you'll use them when you [configure the UID2 mobile SDK](#configure-the-uid2-mobile-sdk).
+
+## Add the UID2 Mobile SDK to Your Mobile App
+
+To add the mobile SDK to your app, follow the applicable documentation:
+
+- [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md)
+- [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md)
+
+At this point, you are ready to start generating UID2 tokens using the SDK.
+
+### Using the UID2 Integration Environment
+
+By default, the SDK is configured to work with the UID2 production environment: `https://prod.uidapi.com`. If you want to use the integration environment instead, provide the following URL in your call to initialize `UID2Manager`:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.init(
+  context = this,
+  serverUrl = "https://operator-integ.uidapi.com"
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+// Must be set before UID2Manager.shared is accessed
+UID2Settings.shared.environment = .custom(
+  url: URL(string: "https://operator-integ.uidapi.com")!
+)
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+Bear in mind the following differences between environments:
+- Tokens from the UID2 integration environment are not valid for passing to the bid stream.
+- You'll have a different set of Subscription ID and public key values for each environment (integ and prod). Be sure to use the correct values for each environment.
+:::
+
+### Optional: Reduce Latency by Setting the API Base URL for the Production Environment
+
+<ReduceLatency />
+
+To specify a different UID2 server, you can change it in the `init` call, as shown in the following examples.
```

Comment:
Reword to:

_To specify a different UID2 server, you can add the following config changes, as shown in the following examples._

iOS code doesn't set such config in `init` call so inconsistent with android's

Reply:
Done, thx.